### ISSUE TITLE
not working on hls fragmented mp4

### ISSUE THREAD

**Issue by siberkolosis** [warren-bank/HLS-Proxy#45]
works, but not as I intended, but rather downloads the whole thing in 1 file. not small pieces like usual hls.
example: https://fediverse.tv/static/streaming-playlists/hls/a26a46d2-f52b-4914-b822-ff6218337d5b/46040db0-0f2a-4fb8-b254-e03c2567e295-master.m3u8

**Comment by warren-bank** [comment]
yeah, I wouldn't recommend using the proxy with this type of manifest.
to be clear, the manifest repeats the URL to the same mp4 file over and over again..
mapping each (4 second) time segment to a byte range within the mp4 file.
